Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+certifi = "*"
```

Comment:
A ton of these are transitive dependencies. Please remove all of those that are. See apertium-streamparser and apertium-apy for guidance.

Reply:
Done

Reply:
Is this why the requirements.txt model is bad?

Reply:
yeah, among other things :)